source/plugins/ruby/in_win_cadvisor_perf.rb

```diff
@@ -50,11 +53,11 @@ def shutdown
     end
 
     def enumerate()
-      time = Time.now.to_f
+      time = Fluent::Engine.now
```

Comment:
how is this change related to removing omsagent for aad auth? instead of getting time.now you are getting from fluentengine, why ?

Reply:
> how is this change related to removing omsagent for aad auth? instead of getting time.now you are getting from fluentengine, why ?

> Yes, this is not related to omsagent and aadauth but related to Fluentd upgrade. Here is the guidance https://docs.fluentd.org/plugin-development/plugin-update-from-v0.12 . Time.now doesnt work with fluentdv1 which explictly and also tested.

Reply:
can you do now=Fluent::Engine.now and use that for every record ?

Reply:
> can you do now=Fluent::Engine.now and use that for every record ?

Sure, I can make that change.